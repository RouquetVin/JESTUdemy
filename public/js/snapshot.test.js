// Snapshot testing...

const examples = require('./script');

// Début de la suite de tests
describe('testing', () => {
	// Test de snapshot
	test('snapshot testing..', () => {
		// Définition de la catégorie à tester
		const category = 'Footwear';

		// Appelle la fonction promotion avec la catégorie "Footwear" et stocke le résultat
		snapshot_rec = examples.promotion(category);

		// Comparaison du résultat actuel avec le snapshot inline attendu
		// Si le résultat diffère, Jest signalera une erreur
		// Pour mettre à jour le snapshot, on peut utiliser "npm run test -- -u"
		expect(snapshot_rec).toMatchInlineSnapshot(
			`"Get 10% discount on all drinkss"`,
		);
	});
});
