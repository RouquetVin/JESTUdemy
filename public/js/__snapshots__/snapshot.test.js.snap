// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing snapshot testing.. 1`] = `"Get 10% discount on all drinks"`;
